api_tests/e2e/rfc003/resume_swap/alice_down_bob_funds.ts

```diff
@@ -0,0 +1,50 @@
+import { sleep } from "../../../lib/util";
+import { Actor } from "../../../lib_sdk/actors/actor";
+import { createActors } from "../../../lib_sdk/create_actors";
+
+setTimeout(function() {
+    let alice: Actor;
+    let bob: Actor;
+
+    beforeEach(async function() {
```

Comment:
I've done some work on the e2e tests recently to make defining a new test less verbose. We should use that everywhere :)

Reply:
This also allows you to move all of the tests into one file, _if you want to do that_.

Reply:
What are the pros/cons of a single file with multiple tests vs a file per test.  In Singapore @luckysori and I played with both and could not think of a strong reason in favour, or against, either.

Reply:
Re-written tests on top of your work @thomaseizinger, all 'resume' tests in a single file and the 'restart' test in its own.

Reply:
> What are the pros/cons of a single file with multiple tests vs a file per test. In Singapore @luckysori and I played with both and could not think of a strong reason in favour, or against, either.

No real benefit in the way we are using mocha.
You can use `describe` blocks to group tests together and have `afterEach` hooks etc. I consider those to be a bad practice because you it makes the tests not isolated. So it doesn't matter in our case.